pkg/minikube/config/config.go

```diff
@@ -150,8 +150,8 @@ func encode(w io.Writer, m MinikubeConfig) error {
 }
 
 // Load loads the Kubernetes and machine config for the current machine
-func Load(profile string) (*ClusterConfig, error) {
-	return DefaultLoader.LoadConfigFromFile(profile)
+func Load(profile string, miniHome ...string) (*ClusterConfig, error) {
```

Comment:
Was this added so that you could pass in `miniHome` for the unit test?

Reply:
yes, as this comment https://github.com/kubernetes/minikube/pull/8533#discussion_r444372072